tools/convertTranslations.js

```diff
@@ -21,15 +21,15 @@ function parseAndroid(data) {
   for (let line of lines) {
     let kv = line.match(rgxKeyValue);
     if (kv != null) {
-      result.parsed.push([kv[1], kv[2].replace(/&quot;/g, '"').
+      result.parsed.push([kv[1], kv[2].replace(/&quot;/g, '\"').
       replace(/&lt;/g, '<').
       replace(/&gt;/g, '>').
       replace(/&amp;/g, '&').
-      replace(/\\t/g, '\t').
-      replace(/\\r/g, '\r').
-      replace(/\\n/g, '\n').
-      replace(/\\\\/g, '\\').
-      replace(/\$s/ig, '$@')])
+      replace(/\t/g, '\t').
+      replace(/\r/g, '\r').
+      replace(/\n/g, '\n').
+      replace(/\\/g, '\\').
+      replace(/$s/ig, '$@')])
```

Comment:
this looks wrong to me. `$` matches end-of-line. i think the escaping is needed here.